src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import NavigationBar from './components/NavigationBar'
+import {Routes, Route} from 'react-router-dom';
+import PrivateRoute from './components/PrivateRoute';
+import Login from './views/Login';
+import ProductListing from './views/Products/ProductListing';
+import Product from './views/Products/Product';
+// import Register from './views/Register';
+import Home from './views/Home';
+import Cart from './views/Cart';
+import Wishlist from './views/Wishlist';
```

Comment:
Seems everything is imported from the views folder.
Please add `index.js` file there to have clean imports here